presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -1008,4 +1010,24 @@ public void testSTGeometryType()
         assertFunction("ST_GeometryType(ST_GeometryFromText('GEOMETRYCOLLECTION(POINT(4 6),LINESTRING(4 6, 7 10))'))", VARCHAR, "ST_GeomCollection");
         assertFunction("ST_GeometryType(ST_Envelope(ST_GeometryFromText('LINESTRING (1 1, 2 2)')))", VARCHAR, "ST_Polygon");
     }
+
+    private void assertToFromBinary(String wkt)
+    {
+        assertToFromBinary(wkt, wkt);
+    }
+
+    private void assertToFromBinary(String wkt, String expected)
+    {
+        assertFunction(format("ST_AsText(ST_GeometryFromBinary(ST_AsBinary(ST_GeometryFromText('%s'))))", wkt), VARCHAR, expected);
+    }
+
+    @Test
+    public void testSTGeometryFromBinary()
+    {
+        assertToFromBinary("POINT EMPTY");
```

Comment:
For consistency, let's rename this function to `assertGeomFromBinary` (assuming the test will be renamed to `testSTGeomFromBinary`).